### PR TITLE
Some refactoring and SPaG fixes

### PR DESCRIPTION
**Pull request by HopperElec**
Edit: Didn't mean to include https://github.com/mlomb/chat-analytics/pull/94 in this, but I'll probably just cause issues with merging if I try to remove it

### CODE REVIEW

**Comment by HopperElec** [comment]
format-check on obfuscator is fair enough, although that syntax is strange and my IDE agrees, but I definitely ran Prettier on Avatars.less and just tried again now and it's supposedly already formatted correctly so I'm not sure what that's about

File: assets/styles/Labels.less [hunk 1]
```diff
@@ -36,8 +36,7 @@
     font-weight: normal;
     color: rgb(255 255 255 / 65%);
     font-family: monospace;
-    margin: auto;
-    margin-left: 3px;
+    margin: auto auto auto 3px;
```
Comment (mlomb) [inline assets/styles/Labels.less:39]: This is in fact shorter, but less readable. I don't usually remember which of the four arguments is what (less when there are three). Right now I see it as "everything with auto but left with 3px", so much easier. This kind of changes are handled by the CSS postprocessor so they have no impact.

There are a few cases of this

File: report/components/viz/amcharts/Distribution.ts [hunk 2]
```diff
@@ -47,9 +47,9 @@ export const createHistogramWithBoxplot = (root: Root, xAxisType: AxisType) => {
         const step = (data.boxplot.whiskerMax - data.boxplot.whiskerMin) / data.count.length;
         let accum = data.boxplot.whiskerMin;
 
-        for (let i = 0; i < data.count.length; i++) {
+        for (const value of data.count) {
             values.push({
-                value: data.count[i],
+                value: value,
```
Comment (mlomb) [inline report/components/viz/amcharts/Distribution.ts:52]: We can change this to just `value`

**Comment by mlomb** [comment]
By the way, what do you mean with "SPaG"? 🤔

**Comment by HopperElec** [comment]
> By the way, what do you mean with "SPaG"? 🤔

Spelling, Punctuation and Grammar. Sorry, probably just a UK thing lol

File: assets/styles/Labels.less (hunk 1 above)
Comment (HopperElec) [reply assets/styles/Labels.less:39]: Fair enough, I didn't know LESS did optimization

Comment (HopperElec) [reply assets/styles/Labels.less:39]: There's quite a few places where you were already using this syntax; for consistency, should I expand those too?

Comment (mlomb) [reply assets/styles/Labels.less:39]: > There's quite a few places where you were already using this syntax; for consistency, should I expand those too?

As you wish

Comment (HopperElec) [reply assets/styles/Labels.less:39]: Well, it's your project, so it should be as you wish, but so be it 😅

**Comment by mlomb** [comment]
I carefully checked all the changes and I'm ready to merge!

-------

I added the following again:
```css
fieldset {
  display: grid;
  display: contents;
}
```
Which was necessary for the Stepper (homepage, while processing) to display correctly.

-------

I rolled back changes in `BitStream`:

```ts
value >>>= 7; // from this
value = value >>> 7; // back to this
```

I don't recall correctly and can't find a source but I had problems with `>>>=` in some browsers (iirc Safari), so I prefer to keep it that way

-------

Thanks for your PR once again! Lot of work, thanks!! 🎊🎊🎊

**Comment by HopperElec** [comment]
Hm I did definitely test the stepper a lot and didn't see any issues, I don't know what having two display properties is supposed to do. Of will. But that older browser issue makes sense, no worries!

**Comment by mlomb** [comment]
> having two display properties is supposed to do

iirc it was a fallback

**Pull request merged**